### PR TITLE
Fixes #2 Adding issue and pull request templates

### LINKED ISSUE #2: Add ISSUE_TEMPLATE.md and PULL_REQUEST_TEMPLATE.md files
Adding template files will make it easier to contribute and so these should be added.

### PR DESCRIPTION
**Pull request by Kerl1310**
Fixes #2 Adding issue and pull request templates

### CODE REVIEW

**Comment by hugovk** [comment]
Tip: add "Fixes #2" to the PR text to automatically close that issue when merged.

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+## Steps to Reproduce
+Please also include relevant information and steps to reproduce the bug/issue.
+
+
+## Expected Functionality
+Explain what should happen.
+
+
+## Experienced Functionality
+Explain what happened instead(Please include the debug log).
```
Comment (hugovk) [inline ISSUE_TEMPLATE.md]:
```suggestion
Explain what happened instead (please include the debug log).
```

**Comment by skullydazed** [comment]
Thanks for submitting these. As with the qmk_compiler PR this should look more like qmk_firmware's: https://github.com/qmk/qmk_firmware/tree/master/.github

**Comment by skullydazed** [comment]
This one as well should be similar to qmk_firmware: https://github.com/qmk/qmk_firmware/tree/master/.github

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ - Keyboard:  
+   - Revision (if applicable):  
+ - Operating system:
+ - AVR GCC version: 
+<!-- Run `avr-gcc --version` to find this out. -->
+ - ARM GCC version: 
+<!-- Run `arm-none-eabi-gcc --version` to find this out. -->
+ - QMK Firmware version:
+<!-- Run `git describe --abbrev=0 --tags` to find this out. -->
+ - Any keyboard related software installed?
```
Comment (hugovk) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
 - Any keyboard-related software installed? 
```

**Comment by Kerl1310** [comment]
> Just a teeny-tiny little suggestion.

Committed change so it's now fixed

**Comment by Kerl1310** [comment]
@hugovk This has been approved almost a month ago and is awaiting merging

**Comment by hugovk** [comment]
Yep, I don't have merge rights, hopefully someone from QMK will check it soon :)

**Comment by noroadsleft** [comment]
Sorry, we've been pretty overwhelmed with things around here, and I don't think this repository gets checked frequently.

There's a lot of [CLI stuff happening over in qmk/qmk_firmware](https://github.com/qmk/qmk_firmware/labels/cli).

**Comment by noroadsleft** [comment]
Additionally, I'm not sure the templates used on qmk_firmware are appropriate for use here, due to the different nature of the projects.

Templates *are* a good thing to have though.

**Pull request closed without merging**